paperqa/docs.py

```diff
@@ -66,6 +66,25 @@ def __init__(
             embeddings = OpenAIEmbeddings()
         self.embeddings = embeddings
 
+        if self._in_notebook():
+            import nest_asyncio
+            nest_asyncio.apply()
+    
+    def _in_notebook(self) -> bool:

```

Comment:
Perhaps this function could be put in `utils.py` (outside of the class), since it doesn't use `self` in the method?